app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -13,7 +13,9 @@ import com.google.android.material.bottomnavigation.BottomNavigationView
 import edu.uc.hickmadc.customnotifications.ui.main.MyListAdapter
 import edu.uc.hickmadc.customnotifications.ui.main.dialogfragment
 
-
+/*
+  Temporary data
```

Comment:
temporary data, it disappears when you click from one tab to another. A technical suggestion here would be to check and see implementing a catch block may be a good idea.

Reply:
This comment isn't adding much value.  What is it describing?